infra.bs

```diff
@@ -103,6 +103,65 @@ specifications using the Infra Standard, the user agent is generally the client
 implements the specification.
 
 
+<h3 id=privacy>Privacy concerns</h3>
+
+<p>Some features that are defined in documents using the Infra Standard might trade user convenience
+for a measure of user privacy.
+
+<p>In general, due to the internet's architecture, a user can be distinguished from another by the
+user's IP address. IP addresses do not perfectly match to a user; as a user moves from device to
+device, or from network to network, their IP address will change; similarly, NAT routing, proxy
+servers, and shared computers enable packets that appear to all come from a single IP address to
+actually map to multiple users. Technologies such as onion routing can be used to further anonymize
+requests so that requests from a single user at one node on the internet appear to come from many
+disparate parts of the network. [[RFC791]]
+
+<p>However, the IP address used for a user's requests is not the only mechanism by which a user's
+requests could be related to each other. Cookies, for example, are designed specifically to enable
+this, and are the basis of most of the web's session features that enable you to log into a site
+with which you have an account. More general, any kind of cache mechanism or shared state, including
+but not limited to HSTS, the HTTP cache, grouping of connections, storage APIs, can and ought to be
+expected to be abused. [[COOKIES]] [[RFC6797]] [[STORAGE]]
+
+<p>There are other mechanisms that are more subtle. Certain characteristics of a user's system can
+be used to distinguish groups of users from each other; by collecting enough such information, an
+individual user's browser's "digital fingerprint" can be computed, which can be as good, if not
```

Comment:
```suggestion
individual user's browser's "digital fingerprint" can be computed, which can be better than an IP address
```

Fingerprints can stably identify an exact device, which is way better than an IP address.